static/js/src/chart-data.js

```diff
@@ -1287,7 +1287,7 @@ export var kubernetesReleases = [
     startDate: new Date("2020-01-07T00:00:00"),
     endDate: new Date("2020-12-07T00:00:00"),
     taskName: "Kubernetes 1.17",
-    status: "CHARMED_KUBERNETES_SUPPORT",
+    status: "CHARMED_KUBERNETES_EXPIRED_SUPPORT",
   },
   {
     startDate: new Date("2020-03-24T00:00:00"),
```

Comment:
Hasn't 1.18 also expired?

Reply:
no. The support is n-2 effectively, so 1.18 is supported until 1.21 is released

Reply:
I will update the small screen table and the values with my best guess for release dates of 1.21. Unfortunately this table is going to get way messier because of changes to the upstream supported versions (they now run for a year instead of 9 months) vs our support (n-2). It is likely our support terms will be updated in the near future to address some of this